### DISCUSSION TITLE
[Question]: local docker install for remote access

### DISCUSSION THREAD

**Discussion by oktaykarakiya** [danny-avila/LibreChat#4048] [Q&A]
### What is your question?

I would like to be able to access my LOCAL docker install on my server, from anywhere, but when i do i get this error in the chat: Something went wrong. Here's the specific error message we encountered: An error occurred while processing your request. Please contact the Admin.

I tried changing the DOMAIN_SERVER = localhost:3080 into my pulblic_ip:3080 but to no avail.

### More Details

I know it would be best to use the guide for remote install, but I am lazy.

### What is the main subject of your question?

Installation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Look into ngrok

**Comment by o42o** [comment]
I've used cloudflared and it works very well.

**Comment by fuegovic** [comment]
https://www.librechat.ai/docs/remote/ngrok
https://www.librechat.ai/docs/remote/cloudflare
